python/tvm/relax/frontend/torch/dynamo.py

```diff
@@ -0,0 +1,178 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# pylint: disable=invalid-name, missing-function-docstring, not-callable
+# pylint: disable=import-outside-toplevel, unused-argument
+# mypy: ignore-errors
+"""PyTorch Dynamo backend of Relax."""
+import functools
+
+import tvm
+from tvm import relax, meta_schedule as ms
+from tvm.relax.vm import build as relax_build
+from tvm.relax.frontend.torch.fx_translator import from_fx
+
+
+def device_from_inputs(example_inputs):
+    for x in example_inputs:
+        if hasattr(x, "device"):
+            return x.device
+    return None
+
+
+def default_dynamo_pipeline(mod: tvm.ir.IRModule, target: tvm.target.Target) -> tvm.ir.IRModule:
+    """A default pipeline for dynamo.
+
+    Parameters
+    ----------
+    mod : tvm.IRModule
+        The relax module to be transformed.
+
+    target : tvm.target.Target
+        The target to be compiled.
+
+    Returns
+    -------
+    mod : tvm.IRModule
+        The transformed relax module.
+    """
+    mod = tvm.transform.Sequential(
```

Comment:
need target and pass context?

Reply:
It looks to me that pass context is something user want to customize inside transform.

Reply:
I now agree with you